src/main/java/org/kainos/ea/resources/AuthController.java

```diff
@@ -0,0 +1,53 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.AuthService;
+import org.kainos.ea.cli.LoginRequest;
+import org.kainos.ea.client.CouldNotGeneratePasswordHashException;
+import org.kainos.ea.client.CouldNotStoreNewJWTException;
+import org.kainos.ea.client.DatabaseConnectionFailedException;
+import org.kainos.ea.client.JWTCouldNotBeCreatedException;
+import org.kainos.ea.client.InvalidLoginAttemptException;
+import org.kainos.ea.client.CouldNotFindUserAccountException;
+import org.kainos.ea.db.AuthDAO;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.security.NoSuchAlgorithmException;
+import java.sql.SQLException;
+
+@Api("Authentication Controller")
+@Path("/api/auth")
+public class AuthController {
+
+  private final AuthService authService = new AuthService();
+  private final AuthDAO authDAO = new AuthDAO();
+
+  @POST
+  @Path("/login")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response userLogin(LoginRequest loginRequest) {
```

Comment:
This endpoint has no validation on the input from the user

Reply:
Added Now. Thank you!